packages/website/src/examples/form/code-editor/CodeEditorXML.demo.tsx

```diff
@@ -0,0 +1,19 @@
+import {CodeEditor, useForm} from '@coveord/plasma-mantine';
+
+export default () => {
+    const form = useForm({
+        initialValues: {
```

Comment:
having the code for the demo is nice but you forgot to plug it in the actual page 😆 

<img src="https://media0.giphy.com/media/uw3fTCTNMbXAk/giphy.gif"/>

Reply:
I didn't forget 😉 Its just that having multiple examples with the new sandbox isn't supported yet. It is in my upcoming PR :hype:. I plan on updating the code editor examples after this particular upcoming PR is merged

Reply:
ohhh noice ❤️‍🔥  I was too eager to see the XML in action it seems 😂